tests/integration-tests/tests/ultraserver/test_gb200/test_gb200/pcluster.config.update.yaml

```diff
@@ -0,0 +1,54 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: c5.xlarge
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  CustomActions:
+    OnNodeStart:
+      Script: s3://{{ bucket_name }}/{{ head_node_start_script }}
+  Iam:
+    S3Access:
+      - BucketName: {{ bucket_name }}
+Scheduling:
+  Scheduler: {{ scheduler }}
+  SlurmQueues:
+  - Name: {{ queue_with_imex }}
+    {% if capacity_block_reservation_id %}
+    CapacityType: CAPACITY_BLOCK
+    CapacityReservationTarget:
+      CapacityReservationId: {{ capacity_block_reservation_id }}
+    {% endif %}
+    ComputeResources:
+      - Name: {{ compute_resource_with_imex }}
+        Instances:
+          - InstanceType: {{ instance }}
+        MinCount: {{ max_queue_size }}
+        MaxCount: {{ max_queue_size }}
+        HealthChecks:
+          Gpu:
+            Enabled: True
+    Networking:
+      SubnetIds:
+        - {{ private_subnet_id }}
+  - Name: {{ queue_without_imex }}
+    ComputeResources:
+      - Name: {{ compute_resource_without_imex }}
+        Instances:
+          - InstanceType: t3.medium
+        MinCount: 1
+        MaxCount: 1
+    Networking:
+      SubnetIds:
+        - {{ private_subnet_id }}
+
+# The DevSetting below forces the configuration of NVIDIA IMEX and topology plugin on instances that are not GB200.
+# We do it to simulate the behavior of GB200 instances.
+{% if instance != "p6e-gb200.36xlarge" %}
+DevSettings:
+  Cookbook:
+    ExtraChefAttributes: |
```

Comment:
In the next iteration, add another update to the test where we remove the force_configuration for Block_topology and see that the whole topology is removed at cluster level. So we should not see any output for `scontrol show topology` and `scontrol show config | grep TopologyPlugin` should be empty